catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/event/SubscriptionManager.java

```diff
@@ -0,0 +1,345 @@
+            }
+        } else {
+            LOGGER.debug("Subscription registration failed");
+        }
+
+        LOGGER.debug("EXITING: " + methodName);
+        return subscriptionUuid;
+
+    }
+
+    public static boolean deleteSubscription(BundleContext context, String subscriptionId)
+            throws InvalidSubscriptionException {
+        String methodName = "deleteSubscription";
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("ENTERING: " + methodName);
+            LOGGER.debug("subscriptionId = " + subscriptionId);
+        }
+
+        if (StringUtils.isEmpty(subscriptionId)) {
+            throw new InvalidSubscriptionException(
+                    "Unable to delete subscription because subscription ID is null or empty");
+        }
+
+        boolean status = false;
+
+        try {
+            if (!StringUtils.isEmpty(subscriptionId)) {
+                LOGGER.debug("Removing (unregistering) subscription: " + subscriptionId);
+                ServiceRegistration sr = (ServiceRegistration) registeredSubscriptions.get(
+                        subscriptionId);
+                if (sr != null) {
+                    sr.unregister();
+                } else {
+                    LOGGER.debug(
+                            "No ServiceRegistration found for subscription: " + subscriptionId);
+                }
+
+                Configuration subscriptionConfig = getSubscriptionConfiguration(context,
+                        subscriptionId);
+                try {
+                    if (subscriptionConfig != null) {
+                        LOGGER.debug(
+                                "Deleting subscription for subscriptionId = " + subscriptionId);
+                        subscriptionConfig.delete();
+
+                        // Subscription removal is only successful if able to remove from OSGi registry and
+                        // ConfigAdmin service
+                        if (sr != null) {
+                            status = true;
+                        }
+                    } else {
+                        LOGGER.debug("subscriptionConfig is NULL for ID = " + subscriptionId);
+                    }
+                } catch (IOException e) {
+                    LOGGER.error(
+                            "IOException trying to delete subscription's configuration for subscription ID "
+                                    + subscriptionId,
+                            e);
+                }
+
+                LOGGER.debug("Removing subscription from internal hash map");
+                registeredSubscriptions.remove(subscriptionId);
+                subscriptions.remove(subscriptionId);
+                LOGGER.info("Subscription removal complete");
+            } else {
+                LOGGER.debug("subscriptionId is NULL or empty");
+            }
+        } catch (Exception e) {
+            LOGGER.info("Could not delete subscription for " + subscriptionId);
```

Comment:
We don't need both the info and the error; put the info's message in place of `e.getMessage()` in the error log.